### ISSUE TITLE
Change the modpack updater to be triggered from the "Version" tab instead of by creating a new instance

### ISSUE THREAD

**Issue by ghost** [PrismLauncher/PrismLauncher#261]
### Role

I use the launcher.

### Suggestion

I want the launcher not to offer one to update an existing instance when a modpack is downloaded and instead add a button to update an instance from a modpack in instance options.

### Benefit

Because the current modpack updater is: 
- annoying 
- does not specify the instance(-es?!) it wants to update 
- can't even be disabled for some reason 
- for whatever reason pressing X on the popup counts as "update the existing instance, i totally do not want to abort the operation" 

This change would make it so the updater is only ever triggered willfully by the user + is beter communicated to them.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by flowln** [comment]
https://github.com/PrismLauncher/PrismLauncher/pull/32

we'll prob remove the existing one once that is ready, i agree it's not very good rn

**Comment by TayouVR** [comment]
imo the existing one can stay, can still be useful. if e.g. someone forgets, that they already have a modpack, being asked to update isn't too bad of a thing to have.

**Comment by ghost** [comment]
> imo the existing one can stay, can still be useful. if e.g. someone forgets, that they already have a modpack, being asked to update isn't too bad of a thing to have.

yes but it'd be annoying in more cases and redundant if a way to properly update a modpack via the instance menu is implemented

**Comment by Scrumplex** [comment]
Duplicate of #180. Will be implemented in #32

**Comment by ghost** [comment]
( @Scrumplex the obsolete updater dialogue is still around as of latest develop)
